### PR TITLE
CLOUDP-141993: Split integration test

### PR DESCRIPTION
**Pull request by Sugar-pack**
### All Submissions:

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

File: .github/workflows/test-int.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test: ["AtlasProject", "AtlasDeployment", "AtlasDatabaseUser"]
+        test: ["AtlasProject","AtlasDatabaseUser", "legacy-deployment", "advanced-deployment"]
```
Comment (igor-karpukhin) [inline .github/workflows/test-int.yml:18]:
```suggestion
        test: ["AtlasProject", "AtlasDatabaseUser", "AtlasLegacyDeployment", "AtlasAdvancedDeployment"]
```

**Review by igor-karpukhin — APPROVED** [review]
Looks good 👍 Please consider my change for tests naming.

File: .github/workflows/test-int.yml (hunk 1 above)
Comment (Sugar-pack) [reply .github/workflows/test-int.yml]: My bad. Now it makes no sense to separate tests

**Review by helderjs — APPROVED** [review]
LGTM

**Pull request merged**